sdk/src/feature_set.rs

```diff
@@ -801,6 +801,10 @@ pub mod abort_on_invalid_curve {
     solana_sdk::declare_id!("FuS3FPfJDKSNot99ECLXtp3rueq36hMNStJkPJwWodLh");
 }
 
+pub mod enable_syscall_get_epoch_stake {
```

Comment:
```suggestion
pub mod enable_get_epoch_stake_syscall {
```
Nit. I know we're definitely not consistent about this, but seems slightly more common to put `syscall` at the end.